docs/administration/Query_planning.md

```diff
@@ -193,6 +193,104 @@ By removing the specific expressions (only keep the operators), the query plan c
 
 ![8-5](../assets/8-5.png)
 
+## Query hint
+
+Query hints are directives or comments that explicitly suggest the query optimizer on how to execute a query. Hints only take effect within a single query.
+
+### System variables hint
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/a8d3f8ad-7f0a-4d6e-9db1-c1391cd49824)
remove "s" in "variables"